test/integration/joint_dom.cc

```diff
@@ -857,3 +859,43 @@ TEST(DOMJoint, LoadJointNestedParentChild)
     EXPECT_EQ(Pose(0, -1, 1, IGN_PI_2, 0, 0), pose);
   }
 }
+
+
+//////////////////////////////////////////////////
+TEST(DOMJoint, Sensors)
+{
+  const std::string testFile =
+      sdf::testing::TestFile("sdf", "joint_sensors.sdf");
```

Comment:
I think this file is missing

Reply:
I always forget to add the sdf file 🤦🏾‍♂️ . Added in https://github.com/osrf/sdformat/pull/517/commits/8d50882e74804f3dc899dc2892ff90b63475446b